examples/todomvc/src/containers/MainSection.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux'
+import * as TodoActions from '../actions'
+import { bindActionCreators } from 'redux'
+import MainSection from '../components/MainSection'
+import { getCompletedTodoCount } from '../selectors'
+
+
+const mapStateToProps = (state) => {
```

Comment:
Seems like in some cases you're using an implicit return for an object and in some cases an explicit one. I would recommend keeping it consistent and using an implicit return in all cases then, unless you require a block for extra statements.